website/common/script/libs/taskDefaults.js

```diff
@@ -65,6 +65,14 @@ module.exports = function taskDefaults (task, user) {
   }
 
   if (task.type === 'daily') {
+    let now = moment().utcOffset(user.preferences.timezoneOffset);
```

Comment:
can you use `.zone` instead? it's deprecated but it's the correct method to use with the data we store